src/keyring.ts

```diff
@@ -18,7 +18,7 @@ import { encodeKey, hexToU8A, u8aToHex } from './utils'
 export const LATEST = 'latest'
 const GENESIS = 'genesis'
 const BASE_PATH = "m/51073068'"
-const ROOT_STORE_PATH = "0'/0'/0'/0'/0'/0'/0'/0'/0'"
+const LEGACY_BASE_PATH = "m/7696500'/0'/0'"
```

Comment:
haha, I bet this is the one that says `muport`. Anyway, this will make things faster 👍